### ISSUE TITLE
Misleading wording on upload page

### ISSUE THREAD

**Issue by rdb** [pyweekorg/pyweekorg#68]
The upload page says the following:
>    You may mark as many files "final" as you like - the very last one will
>    be used in the peer judging.

This is misleading, because the reasonable interpretation is that only one file should be marked as "final".  However, all files marked "final" will appear on the Entries page, and can therefore be used for judging.

**Comment by lordmauve** [comment]
Yeah, I think this is a vestigial message from the prehistoric era of PyWeek.
